pkg/storaged/mdraid-details.jsx

```diff
@@ -208,6 +191,23 @@ export class MDRaidDetails extends React.Component {
             );
         }
 
+        function fix_bitmap() {
+            return mdraid.SetBitmapLocation(utils.encode_filename("internal"), { });
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18979-20230630-121722-3b196e14-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)